web_accounts_list.json

```diff
@@ -4436,6 +4436,72 @@
           "known_accounts" : ["tata23", "fitzsta"],
           "category" : "XXXPORNXXX",
           "valid" : true
-        }
+        },
+        {
+          "name" : "Snapchat",

```

Comment:
We already have a Snapchat entry. Can you change this to a name of "Snapchat Story" please since it is another valid detection string?

Reply:
Absolutely! I should do a better job of double-checking beforehand but I guess having the alternate detection method isn't a bad thing, eh?